test/unit/SwapOperator/swapETHForEnzymeVaultShare.js

```diff
@@ -0,0 +1,346 @@
+const { ethers } = require('hardhat');
+const { expect } = require('chai');
+const { time } = require('@openzeppelin/test-helpers');
```

Comment:
pls don't add new usages for `openzeppelin/test-helpers`, as we're deprecating it